src/main/resources/schema.sql

```diff
@@ -0,0 +1,47 @@
+DROP DATABASE IF EXISTS `ah_blacklist`;
```

Comment:
use the same method we use in SR.  create_empty_db, create_tables, grant_privileges